.github/workflows/acceptance-tests.yml

```diff
@@ -1,31 +1,58 @@
 name: Acceptance Tests
 
 on:
-  schedule:
-    - cron: "30 7 * * *"
-  workflow_dispatch:
+  issue_comment:
+    types: [created]
```

Comment:
I'm guessing this is to be triggered when a comment is created

Reply:
yes, the idea was to have it triggered if a member of `@aiven/aiven-terraform-reviewers` adds a `/acceptance_tests` comment on a PR

Reply:
:+1: